pkgs/development/web/postman/darwin.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenvNoCC
+, fetchurl
+, unzip
+, pname
+, version
+, meta
+, ...
```

Comment:
```suggestion
```

Reply:
When do you use the _three dots_?

Reply:
It's sort of a catch-all for unexpected attributes.

Using it in a shell or such is fine, but for expressions added to the `nixpkgs` tree, we want to be explicit, so we don't want to use `...` here.

See [this](https://nixos.wiki/wiki/Nix_Expression_Language#Accepting_unexpected_attributes_in_argument_set) page on the Wiki for more information.

Reply:
Great, thanks for the link!